include/swift/AST/DiagnosticsSIL.def

```diff
@@ -476,6 +476,11 @@ NOTE(autodiff_cannot_differentiate_through_inout_arguments,none,
      "cannot differentiate through 'inout' arguments", ())
 NOTE(autodiff_class_member_not_supported,none,
      "differentiating class members is not yet supported", ())
+// TODO(TF-642): Remove when `partial_apply` works with `@differentiable`
+// functions.
+NOTE(autodiff_cannot_index_subset_thunk_partial_applied_orig_fn,none,
+     "cannot convert a direct method reference to a `@differentiable` "
+     "function; please use an explicit closure instead", ())
```

Comment:
```suggestion
     "function; use an explicit closure instead", ())
```
The only Swift diagnostic that uses "please" is "please file a bug report". So, there's no need to use it.

Reply:
Done in https://github.com/apple/swift/pull/26292/commits/6e170f2228ce1f0e13b3e1d5dc988d4f8a82580f.